extensions/ql-vscode/src/skeleton-query.ts

```diff
@@ -30,13 +31,14 @@ export class SkeletonQueryWizard {
 
   constructor(
     private readonly cliServer: CodeQLCliServer,
-    private readonly storagePath: string | undefined,
     private readonly progress: ProgressCallback,
     private readonly credentials: Credentials | undefined,
     private readonly extLogger: OutputChannelLogger,
     private readonly databaseManager: DatabaseManager,
     private readonly token: CancellationToken,
-  ) {}
+  ) {
+    this.storagePath = this.workoutStoragePath();
```

Comment:
It always seems fishy to me when we're calling instance methods in the constructor. The method doesn't depend on any instance variables, so can we move this out of the class and into a function?